student-software-developers/administration-handbook/recruitment.md

```diff
@@ -0,0 +1,4 @@
+### Background on CS departments and advertising strategies
+
+* There are 2 ways to major in Computer Science at Princeton. You can apply and matriculate to the School of Engineering and Applied Science, or you can do the major through the standard BA program. The Undergraduate Affairs Administrator for the School of Engineering, [Traci Miller](https://engineering.princeton.edu/engage/leadership-and-staff), has also posted our position to the Princeton engineering student job listserv in the past. However, an email from Colleen should reach those students as well.
```

Comment:
This is a little confusing since Colleen is mentioned on a different page. Maybe we should move that here and reference it from the january page.